prime-router/docs/proposals/0003-lineage/cool-lineage-example.sql

```diff
@@ -0,0 +1,54 @@
+-- Steps to use this as a demo (a hack, I know)
```

Comment:
I wrote some Postgres stored procedures to show how to meet the key lineage use cases as outlined in the proposal - they are in the next file, lineage-report-stored-procedures.sql 

This example file here, shows actual calls to those stored procs.